taxonomies/categories.txt

```diff
@@ -32296,9 +32296,18 @@ nl:Drinkyoghurts
 
 <en:Drinkable yogurts
 fr:Yaourts à boire goût fraise, Yaourts à boire fraise, Yaourts à boire saveur fraise
+en:Dairy drink with sugar and fruits, Fermented milk with sugar and fruits, Yogurt with sugar and fruits
+fr:Boisson lactée sucrée au lait fermenté et aux fruits, Yaourt à boire sucré aux fruits, Lait fermenté sucré aux fruits
+ciqual_food_code:en:19535
+ciqual_food_name:en:Dairy drink or fermented milk or yogurt, with fruits, with sugar
```

Comment:
One issue with yogurts and drinkable yogurts is that we did not make the distinction between fruits and fruits flavors as CIQUAL did.
So "Yaourts à boire goût fraise" currently can contain both. I'm not sure what to do. cc @moon-rabbitOFF